tests/display.rs

```diff
@@ -350,5 +350,5 @@ fn debug_string() {
     use glium::backend::Facade;
 
     let display = support::build_display();
-    display.get_context().insert_debug_marker("Hello world");
+    display.get_context().insert_debug_marker("Hello world").unwrap();
```

Comment:
The `insert_debug_marker` function isn't guaranteed to work (and it doesn't work on travis/circle-ci).
You should put `display.get_context().insert_debug_marker("Hello world").ok();` instead.